### PR TITLE
Update MVC dependencies

### PR DESCRIPTION
**Pull request by JamesNK**
Update MVC dependencies to react to routing name change in https://github.com/aspnet/AspNetCore/pull/4416

### CODE REVIEW

File: src/Mvc/build/dependencies.props [hunk 1]
```diff
@@ -10,91 +10,91 @@
   
     <AngleSharpPackageVersion>0.9.9</AngleSharpPackageVersion>
     <BenchmarkDotNetPackageVersion>0.10.13</BenchmarkDotNetPackageVersion>
-    <BenchmarksOnlyMicrosoftEntityFrameworkCoreDesignPackageVersion>3.0.0-preview.18572.1</BenchmarksOnlyMicrosoftEntityFrameworkCoreDesignPackageVersion>
-    <BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlitePackageVersion>3.0.0-preview.18572.1</BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlitePackageVersion>
-    <BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlServerPackageVersion>3.0.0-preview.18572.1</BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlServerPackageVersion>
-    <BenchmarksOnlyMySqlConnectorPackageVersion>0.47.1</BenchmarksOnlyMySqlConnectorPackageVersion>
+    <BenchmarksOnlyMicrosoftEntityFrameworkCoreDesignPackageVersion>2.1.1</BenchmarksOnlyMicrosoftEntityFrameworkCoreDesignPackageVersion>
```
Comment (NTaylorMullen) [inline src/Mvc/build/dependencies.props]: This seems wrong. Going from 3.0.0 - 2.1.1? @sebastienros  expected?

Comment (sebastienros) [reply src/Mvc/build/dependencies.props]: We should not change these `BenchmarksOnly` versions. Unless it causes an issue?

Comment (JamesNK) [reply src/Mvc/build/dependencies.props]: These values were automatically updated by running `./run.cmd -Path src/Mvc/ -LockFile ./korebuild-lock.txt upgrade deps`

@natemcmaster @ryanbrandenburg @sebastienros Is the upgrade deps tool putting the wrong values here?

Comment (sebastienros) [reply src/Mvc/build/dependencies.props]: I will look at the script, maybe that's what I should have changed, didn't know it was updating these also. Just revert these lines in the meantime.

**Comment by natemcmaster** [comment]
We're adding required PR checks to ensure the repo compiles before you merge. Can you update this PR by rebasing on master?

```
git fetch origin master:master
git rebase master
```

Alternative: if you plan to 'squash merge' this PR, you can also use a merge to update this PR. This can be easier to manage if you have lots of changes in this PR.
```
git fetch origin master:master
git rebase master
```

File: src/Mvc/build/dependencies.props (hunk 1 above)
Comment (JamesNK) [reply src/Mvc/build/dependencies.props]: Done

**Comment by JamesNK** [comment]
Done

File: src/Mvc/build/dependencies.props [hunk 2]
```diff
@@ -16,85 +16,85 @@
...
+    <MicrosoftExtensionsPropertyActivatorSourcesPackageVersion>3.0.0-preview.18605.1</MicrosoftExtensionsPropertyActivatorSourcesPackageVersion>
+    <MicrosoftExtensionsPropertyHelperSourcesPackageVersion>3.0.0-preview.18605.1</MicrosoftExtensionsPropertyHelperSourcesPackageVersion>
+    <MicrosoftExtensionsSecurityHelperSourcesPackageVersion>3.0.0-preview.18605.1</MicrosoftExtensionsSecurityHelperSourcesPackageVersion>
+    <MicrosoftExtensionsTypeNameHelperSourcesPackageVersion>3.0.0-preview.18605.1</MicrosoftExtensionsTypeNameHelperSourcesPackageVersion>
+    <MicrosoftExtensionsValueStopwatchSourcesPackageVersion>3.0.0-preview.18605.1</MicrosoftExtensionsValueStopwatchSourcesPackageVersion>
+    <MicrosoftExtensionsWebEncodersPackageVersion>3.0.0-preview.18605.1</MicrosoftExtensionsWebEncodersPackageVersion>
     <MicrosoftNETCoreApp20PackageVersion>2.0.9</MicrosoftNETCoreApp20PackageVersion>
     <MicrosoftNETCoreApp21PackageVersion>2.1.3</MicrosoftNETCoreApp21PackageVersion>
     <MicrosoftNETCoreAppPackageVersion>3.0.0-preview-27122-01</MicrosoftNETCoreAppPackageVersion>
```
Comment (dougbu) [inline src/Mvc/build/dependencies.props]: Bit surprised there isn't a newer Microsoft.NetCore.App package. @natemcmaster is the lineup out-of-date?

**Review by dougbu — APPROVED** [review]

**Review by NTaylorMullen — APPROVED** [review]

**Pull request merged**